webapp/src/pages/Dashboard.tsx

```diff
@@ -69,11 +69,11 @@ const Dashboard = () => {
       </Box>
       {datasetInfo?.availableDatasetSplits.train && (
         <PreviewCard
-          title="Dataset Class Distribution Analysis"
-          to={`/${jobId}/dataset_class_distribution_analysis${searchString}`}
+          title="Dataset Warnings"
+          to={`/${jobId}/dataset_warnings${searchString}`}
           description={
             <Description
-              text="Compare the class distribution of your training and evaluation sets."
+              text="Detect issues related to class size, class imbalance or dataset shift between your training and evaluation sets."
```

Comment:
I'm not sure why, but I find "Detect" weird. Detection feels more like a quick check that a computer makes. I liked the "Investigate" you used on the page. Can we reuse it? Also, Grammarly recommends adding a comma.
```suggestion
              text="Investigate issues related to class size, class imbalance, or dataset shift between your training and evaluation sets."
```